CMake/WrappingConfigCommon.cmake

```diff
@@ -1,3 +1,36 @@
+# The default path when not wrapping.  Restore standard build location
+# if python wrapping is turned on, and then turned off.
+if(NOT CMAKE_LIBRARY_OUTPUT_DIRECTORY)
+  set(
+    NO_WRAP_CMAKE_LIBRARY_OUTPUT_DIRECTORY
+    ${ITK_BINARY_DIR}/${CMAKE_INSTALL_LIBDIR}
+    CACHE PATH
+    "Shared library directory"
+  )
+else()
+  set(
+    NO_WRAP_CMAKE_LIBRARY_OUTPUT_DIRECTORY
+    ${CMAKE_LIBRARY_OUTPUT_DIRECTORY}
+    CACHE PATH
+    "Shared library directory"
+  )
+endif()
+if(NOT CMAKE_RUNTIME_OUTPUT_DIRECTORY)
+  set(
+    NO_WRAP_CMAKE_RUNTIME_OUTPUT_DIRECTORY
+    ${ITK_BINARY_DIR}/bin
+    CACHE PATH
+    "Shared library directory"
+  )
+else()
+  set(
+    NO_WRAP_CMAKE_RUNTIME_OUTPUT_DIRECTORY
+    ${CMAKE_RUNTIME_OUTPUT_DIRECTORY}
+    CACHE PATH
+    "Shared library directory"
+  )
+endif()
```

Comment:
`NO_WRAP_CMAKE_RUNTIME_OUTPUT_DIRECTORY` is always defined at this point.

Reply:
By moving these to CMakeWrappingConfigCommon.cmake, it changes the output directory path for both

1. ITK itself
2. External modules

1.) was not the case before. Should be checked:

1. Build without wrapping. List file locations.
2. Enable wrapping. Build.
3. Disable wrapping. Build and List file locations.
4. The file locations in 1.) and 3.) are the same.